excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/GameFieldImplTest.java

```diff
@@ -59,364 +53,380 @@ public void setup() {
         dice = mock(Dice.class);
     }
 
-    private void dice(int...ints) {
+    private void dice(int... ints) {
         OngoingStubbing<Integer> when = when(dice.next(anyInt()));
         for (int i : ints) {
             when = when.thenReturn(i);
         }
     }
 
-    private void givenFl(String board, int fieldHeight) {
-        MapParserImpl level = new MapParserImpl(board, fieldHeight);
-        Hero hero = level.getHeroes().get(0);
+    private void givenFl(String board) {
+        MapParserImpl parser = new MapParserImpl(board);
+        Bike bike = parser.getBikes().get(0);
 
-        game = new GameFieldImpl(level, dice);
+        game = new GameFieldImpl(parser, dice);
         listener = mock(EventListener.class);
         player = new Player(listener);
         game.newGame(player);
-        player.setHero(hero);
-        hero.init(game);
-        this.hero = game.getHeroes().get(0);
+        player.setHero(bike);
+        bike.init(game);
+        this.bike = game.getBikes().get(0);
     }
 
     private void assertE(String expected) {
         assertEquals(TestUtils.injectN(expected),
                 printer.getPrinter(game.reader(), player).print());
     }
 
-    // есть карта со мной
     @Test
     public void shouldFieldAtStart() {
```

Comment:
Let's use given/when/then approach please, because such stuff like was written here before is not so readable.

Reply:
Edited all tests for using given/when/then approach